delfin/drivers/fake_storage/__init__.py

```diff
@@ -521,7 +523,7 @@ def get_capabilities(context):
                         "description": "Average time taken for an IO "
                                        "operation in ms"
                     },
-                    "requests": {
+                    "iops": {
```

Comment:
This changes are merged using https://github.com/sodafoundation/delfin/pull/609 you may rebase

Reply:
Rebased. Thanks